server/athenian/api/models/state/models.py

```diff
@@ -325,6 +327,7 @@ class MappedJIRAIdentity(create_time_mixin(created_at=True, updated_at=True), Ba
     """JIRA identity mapping."""
 
     __tablename__ = "jira_identity_mapping"
+    __table_args__ = (Index("jira_identity_mapping_jira_user_id", "account_id", "jira_user_id"),)
```

Comment:
No-no, indexes are too complex and advanced to mention in the models. We should ignore their existence.

Reply:
I don't understand, should we only manually create them in migrations ? Do we have examples of indexes we were not able to declare in models ? We already have indexes on single columns, unique constraints, check constraints with a custom expression in the models.

Reply:
I missed the INCLUDE